src/js/ripple/orderbook.js

```diff
@@ -532,79 +659,139 @@ OrderBook.prototype.resetOwnerOfferTotal = function(account) {
 };
 
 /**
- * Casts and returns offer's taker gets funded amount as a default IOU amount
+ * Clear owner funds leftovers
+ */
+
+OrderBook.prototype.clearOwnerFundsLeftover = function() {
+  this._ownerFundsLeftover = {};
+};
+
+/**
+ * Reset owner funds leftovers for an account to 0
+ */
+
+OrderBook.prototype.resetOwnerFundsLeftover = function(account) {
+  assert(UInt160.is_valid(account), 'Account is invalid');
+
+  this._ownerFundsLeftover[account] = Amount.from_json('0');
+
+  return this._ownerFundsLeftover[account];
+};
+
+/**
+ * Retrieve leftover funds found after clamping leg one by account
  *
- * @param {Object} offer
- * @return {Amount}
+ * @param [String] account
+ *
+ * @return [Amount]
  */
 
-OrderBook.prototype.getOfferTakerGetsFunded = function(offer) {
-  assertValidNumber(offer.taker_gets_funded, 'Taker gets funded is invalid');
+OrderBook.prototype.getLeftoverOwnerFunds = function(account) {
+  assert(UInt160.is_valid(account), 'Account is invalid');
 
-  return Amount.from_json(offer.taker_gets_funded + OrderBook.IOU_SUFFIX);
+  var amount = this._ownerFundsLeftover[account];
+
+  if (!amount) {
+    amount = Amount.from_json('0' + OrderBook.IOU_SUFFIX);
```

Comment:
"Fix" may be the wrong word, but yeah, sometime the improvement which was suggested previously ought to be made, doesn't have to be part of this pull request